src/topp/MSGFPlusAdapter.cpp

```diff
@@ -194,6 +197,10 @@ class MSGFPlusAdapter :
     setValidStrings_("fixed_modifications", all_mods);
     registerStringList_("variable_modifications", "<mods>", vector<String>(), "Variable modifications, specified using UniMod (www.unimod.org) terms, e.g. 'Oxidation (M)'", false);
     setValidStrings_("variable_modifications", all_mods);
+    
+    registerStringOption_("legacy_conversion", "<choice>", "false", "MS-GF+ search results are available as mzid or tsv. With legacy conversion switched on, the tsv are being used instead of the mzid", false, true);
```

Comment:
@mwalzer The new flag is set to `false` by default (as I want it to be). But does `TOPPBase` also allow for flags which are set `true` by default?